### PR TITLE
Update strings.po all languages 

### PR DESCRIPTION
**Pull request by icarok99**

### CODE REVIEW

File: language/resource.language.uk_ua/strings.po [hunk 1]
```diff
@@ -1648,16 +1648,31 @@ msgstr "Для перегляду інших композицій цього в
...
+msgstr "Індикатор фокусу групи віджетів"
 
 msgctxt "#31348"
 msgid "Press down for channel EPG"
 msgstr "Для перегляду телегіду натисніть вниз"
 
+#: /1080i/SkinSettings.xml
+msgctxt "#31349"
+msgid "Replace close button with menu"
+msgstr ""
```
Comment (dm1tz) [inline language/resource.language.uk_ua/strings.po]: msgstr "Замінити кнопку закриття на меню"

File: language/resource.language.uk_ua/strings.po [hunk 2]
```diff
@@ -1648,16 +1648,31 @@ msgstr "Для перегляду інших композицій цього в
...
+
 msgctxt "#31350"
 msgid "Thanks"
 msgstr "Подяка"
 
+#: /1080i/SkinSettings.xml
+msgctxt "#31351"
+msgid "Hubs"
+msgstr ""
```
Comment (dm1tz) [inline language/resource.language.uk_ua/strings.po]: msgstr "Хаби"

File: language/resource.language.uk_ua/strings.po [hunk 3]
```diff
@@ -1648,16 +1648,31 @@ msgstr "Для перегляду інших композицій цього в
...
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31352"
+msgid "Topbar"
+msgstr ""
```
Comment (dm1tz) [inline language/resource.language.uk_ua/strings.po]: msgstr "Верхнє меню"

File: language/resource.language.uk_ua/strings.po [hunk 4]
```diff
@@ -1687,6 +1702,11 @@ msgctxt "#31358"
 msgid "Waiting for plugins"
 msgstr "Запуск плагінів"
 
+#: /shortcuts/generator/data/setup/powermainmenu_item.xml
+msgctxt "#31359"
+msgid "Switch user"
+msgstr ""
```
Comment (dm1tz) [inline language/resource.language.uk_ua/strings.po]: msgstr "Змінити профіль"

File: language/resource.language.uk_ua/strings.po [hunk 5]
```diff
@@ -1715,11 +1735,21 @@ msgctxt "#31365"
...
+
 #: /1080i/SkinSettings.xml
 msgctxt "#31367"
 msgid "Episode thumb for info dialog background"
 msgstr "Превью серії для фону вікна інформації"
 
+#: /1080i/Includes_Search_View.xml
+msgctxt "#31368"
+msgid "Searching for"
+msgstr ""
```
Comment (dm1tz) [inline language/resource.language.uk_ua/strings.po]: msgstr "Шукаємо"

**Review by dm1tz — APPROVED** [review]
Thanks, looks good except 31349. Please change to msgstr "Замінити кнопку закриття на меню"

**Comment by icarok99** [comment]
> Thanks, looks good except 31349. Please change to msgstr "Замінити кнопку закриття на меню"

Fixed.

**Review by dm1tz — APPROVED** [review]

**Comment by icarok99** [comment]
@gtkingbuild There were some things I didn't edit because they were correct.

**Comment by gtkingbuild** [comment]
Ok, no problem.

File: language/resource.language.fr_fr/strings.po [hunk 6]
```diff
@@ -1654,10 +1654,25 @@ msgctxt "#31348"
 msgid "Press down for channel EPG"
 msgstr "Appuyer vers le bas pour accéder à programme de la chaîne"
 
+#: /1080i/SkinSettings.xml
+msgctxt "#31349"
+msgid "Replace close button with menu"
+msgstr ""
```
Comment (ludovik35) [inline language/resource.language.fr_fr/strings.po]: msgstr "Remplacer bouton fermeture par bouton menu"

File: language/resource.language.fr_fr/strings.po [hunk 7]
```diff
@@ -1654,10 +1654,25 @@ msgctxt "#31348"
...
+
 msgctxt "#31350"
 msgid "Thanks"
 msgstr "Merci"
 
+#: /1080i/SkinSettings.xml
+msgctxt "#31351"
+msgid "Hubs"
+msgstr ""
```
Comment (ludovik35) [inline language/resource.language.fr_fr/strings.po]: msgstr "Hubs"

File: language/resource.language.fr_fr/strings.po [hunk 8]
```diff
@@ -1687,6 +1702,11 @@ msgctxt "#31358"
 msgid "Waiting for plugins"
 msgstr "En attente des extensions"
 
+#: /shortcuts/generator/data/setup/powermainmenu_item.xml
+msgctxt "#31359"
+msgid "Switch user"
+msgstr ""
```
Comment (ludovik35) [inline language/resource.language.fr_fr/strings.po]: msgstr "Changer d'utilisateur"

File: language/resource.language.fr_fr/strings.po [hunk 9]
```diff
@@ -1715,11 +1735,21 @@ msgctxt "#31365"
 msgid "Node"
 msgstr "Noeud"
 
+#: /1080i/Includes_Labels.xml
+msgctxt "#31366"
+msgid "Uncropped"
+msgstr ""
```
Comment (ludovik35) [inline language/resource.language.fr_fr/strings.po]: msgstr "Non rogné"

File: language/resource.language.fr_fr/strings.po [hunk 10]
```diff
@@ -1715,11 +1735,21 @@ msgctxt "#31365"
...
+
 #: /1080i/SkinSettings.xml
 msgctxt "#31367"
 msgid "Episode thumb for info dialog background"
 msgstr "Vignette de l'épisode pour l'arrière-plan de la fenêtre d'information"
 
+#: /1080i/Includes_Search_View.xml
+msgctxt "#31368"
+msgid "Searching for"
+msgstr ""
```
Comment (ludovik35) [inline language/resource.language.fr_fr/strings.po]: msgstr "Recherche de"

**Review by ludovik35 — CHANGES_REQUESTED** [review]
Hi @icarok99,
I'm trying review in your pull.
Tell me if it's ok for you or if you prefer separate pull. 
All options could be ok for me

**Comment by icarok99** [comment]
@ludovik35 add, thanks.

**Comment by icarok99** [comment]
I opened another pull request to stay organized.

**Pull request closed without merging**